rootfs/etc/profile.d/motd.sh

```
@@ -1,9 +1,11 @@
-if [[ $SHLVL -eq 1 ]]; then
+if [[ $SHLVL -eq 1 ]] && ! [[ $GEODESIC_MOTD_ENABLED == "false" ]]; then
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)